chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.ts

```diff
@@ -15,12 +15,15 @@ export class InfoGenerateComponent implements OnInit {
   }
 
   ngOnInit() {
-    // Capture the showSubNavBar if available
-    // tslint:disable-next-line:max-line-length
-    this.route.queryParamMap.pipe(map(params => params.get('showSubNavBar').toLowerCase() === 'true' ? true : false)).subscribe(subNavBarVisible => {
-      this.subNavBarService.setSubNavBarVisibility(subNavBarVisible);
-    });
-  }
 
-}
+    if (this.route.snapshot.queryParamMap.get('showSubNavBar') === null) {
+      stop();
```

Comment:
What does this `stop()` method do?

Reply:
The stop() method works in the same vain as a break within an if statement, thus preventing it from going into the last part of the function

Reply:
I found it within the DOM library but I'm replacing it with 'return'

Reply:
Okay thanks.